pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -874,6 +882,15 @@ func GetBootstrapClusterRoleBindings() []authorizationapi.ClusterRoleBinding {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: OAuthSelfClientAuthorizerRoleBindingName,
+			},
+			RoleRef: kapi.ObjectReference{
+				Name: OAuthSelfClientAuthorizerRoleName,
+			},
+			Subjects: []kapi.ObjectReference{{Kind: authorizationapi.SystemGroupKind, Name: AuthenticatedGroup}},
```

Comment:
and if we did have a role for this, we'd only bind it to the `system:authenticated:oauth` group